core/dive.c

```diff
@@ -205,7 +205,7 @@ void remove_event(struct event *event)
  * have to actually remove the existing event and replace it with a new one.
  * WARNING, WARNING... this may end up freeing event in case that event is indeed
  * WARNING, WARNING... part of this divecomputer on this dive! */
-void update_event_name(struct dive *d, struct event *event, char *name)
+void update_event_name(struct dive *d, struct event *event, const char *name)
```

Comment:
perhaps this `const` qualifier change and the one for `dive_remove_picture()` should have been in a separate commit?

Reply:
Yes, this shouldn't be mixed in with the rest - or you mention in the commit message why you need this change together with the qUtf8Printable() change